apple/internal/resource_rules.bzl

```diff
@@ -217,6 +217,7 @@ def _apple_resource_group_impl(ctx):
                 structured_files,
                 parent_dir_param = partial.make(
                     resources.structured_resources_parent_dir,
+                    parent_dir = "",
```

Comment:
could you instead change it in `apple/internal/resources.bzl` so that `parent_dir` in `_structured_resources_parent_dir` has a default of `None`?

Reply:
the implementation of that method considered a None value, but we didn't put it in the API contract

Reply:
Done

Reply:
> could you instead change it in `apple/internal/resources.bzl` so that `parent_dir` in `_structured_resources_parent_dir` has a default of `None`?

done